Source/WebKit/UIProcess/Cocoa/VideoFullscreenManagerProxy.mm

```diff
@@ -1137,6 +1144,18 @@ - (BOOL)prefersStatusBarHidden
     m_page->send(Messages::VideoFullscreenManager::FullscreenMayReturnToInline(contextId, m_page->isViewVisible()));
 }
 
+void VideoFullscreenManagerProxy::callAfterBoundsHaveChanged(CompletionHandler<void()>&& callback)
+{
+    m_boundsHaveChangedCallbacks.append(WTFMove(callback));
+}
+
+void VideoFullscreenManagerProxy::didResolveBounds()
```

Comment:
I feel like I might need to call this in some destructor to make sure the callbacks always get invoked, but not sure where